src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/CovariantEquals/newline/Test.java

```diff
@@ -16,3 +16,9 @@ public boolean equals(TreeWalker.coding.CovariantEquals.Test i) {  // violation
 //        return false;
 //    }
 }
+
+class Test3 {
+    public boolean equals(TreeWalker.coding.CovariantEquals.Test i) {  // violation without filter
```

Comment:
why this violation is not expected in context/expected.txt? the check is in neverSuppressedChecks

Reply:
@baratali  this is newline/Test.java not context/Test.java

Reply:
ok, I thought all inputs are the same.